datasets/us/md/med_exclusions/crawler.py

```diff
@@ -0,0 +1,76 @@
+from typing import Dict
+from rigour.mime.types import XLSX
+from openpyxl import load_workbook
+
+from zavod import Context, helpers as h
+
+SANCTION_TYPES = {
+    "MCFU-A": "Medicaid Fraud Control Unit Adult Abuse",
+    "LB": "Licensing Board",
+    "HD": "Health Department",
+    "HHS": "Department of Health and Human Services Medicaid Medicare Services",
+    "F": "Individual or Entity Convicted of Medicaid Fraud",
+}
+
+
+def crawl_item(row: Dict[str, str], context: Context):
+
+    if not row.get("first_name"):
+        entity = context.make("Company")
+        entity.id = context.make_id(row.get("last_name_organization"))
+        entity.add("name", row.pop("last_name_organization"))
+    else:
+        entity = context.make("Person")
+        entity.id = context.make_id(
+            row.get("last_name_organization"), row.get("first_name")
+        )
+        h.apply_name(
+            entity,
+            first_name=row.pop("first_name"),
+            last_name=row.pop("last_name_organization"),
+        )
+
+    entity.add("npiCode", row.pop("npi"))
+    entity.add("sector", row.pop("type_of_entity_profession"))
+    entity.add("topics", "debarment")
+    entity.add("country", "us")
+
+    if row.get("license_no"):
+        entity.add("description", "License No: " + row.pop("license_no"))
+
+    if row.get("address") or row.get("city_state_zip"):
+        address = h.make_address(
+            context,
+            (row.pop("address") or "") + " " + (row.pop("city_state_zip") or ""),
+        )
+        entity.add("address", address)
```

Comment:
watch out, `Address` entities must be added to the `addressEntity` property. Address `string`s are added to the `address` property.

So for address strings like this, merely adding to `address` prop is good.

We use `Address` entities when we have more granular address data like a separate zip code and/or state. But parsing addresses like this into their parts is out of scope at the moment.